File: Function/5.py
input_id = input("아이디를 입력해주세요 : ")

def login(_id) :
    members = ['DDangNon','Non','test']
    for member in members :
        if member == _id :
            return True
    return False

if login(input_id) :
    print('Hello ' + input_id)
else :
    print('who are you')

File: Function/5.rb
puts("아이디를 입력해주세여")
input_id = gets.chomp()

def login(id)
    members = ['DDangNon','Non','bloger']
    for member in members do
        if member == id
            return true
        end
    end
    return false
end

if login(input_id)
    puts('Hello, ' + input_id)
else
    puts('Who are you')
end